docs/content/preview/yugabyte-platform/manage-deployments/xcluster-replication/bidirectional-replication.md

```diff
@@ -0,0 +1,91 @@
+- You can't use the YugabyteDB Anywhere UI to create two separate replication configurations for YSQL, each containing a subset of the database tables.
+
+For more information, refer to [Limitations](../../../../architecture/docdb-replication/async-replication/#limitations).
+
+## Prerequisites
+
+- Create two universes as described in [Prerequisites](../xcluster-replication-setup/#prerequisites).
+
+## Set up bidirectional replication
+
+To set up bidirectional replication from universe A to universe B, and vice versa on the same set of tables or databases, do the following:
+
+1. Make sure that there are no writes happening on universe B.
+1. Set up xCluster replication from A to B following the steps in [Deploy xCluster](../../../../deploy/multi-dc/async-replication/async-deployment/). A full copy of the selected tables or databases may be performed as part of this operation.
+1. Set up xCluster replication from universe B to A following the steps in [Deploy xCluster](../../../../deploy/multi-dc/async-replication/async-deployment/).
+
+    No full copy is performed, so it is important to have no writes on B during this setup, as these writes might not be replicated to universe A.
+
+For information on how to monitor and manage the replication configurations, refer to [Monitor replication](../xcluster-replication-setup/#monitor-replication).
+
+## Restart bidirectional replication
+
+Restarting bidirectional replication can cause unreplicated data on one of the universes to be lost, because a full copy is performed as part of the restart. Copying data can be done only if an xCluster configuration with reverse direction for a table does not exist, which means the reverse replication must first be deleted.
+
+As a result, before restarting bidirectional replication, you need to identify the universe whose data is more up to date.
+
+To restart a bidirectional replication setup:
+
+1. Identify the universe whose data is more up to date.
+1. Delete the replication configuration where this universe is the target universe.
+1. Restart the other replication configuration, where this universe is the source universe.
+1. Set up the replication configuration deleted in step (2) again, with this universe as the target universe.
+
+Note that restarting one side of an existing bidirectional replication configuration without performing step 2 simply re-establishes the replication configuration from the current data without performing a full copy. This can cause inconsistencies in the data between the source and target universes.
+
+## DDL operations in bidirectional replication
+
+In general, for DDL operations, you can follow the steps in [Manage tables and indexes](../xcluster-replication-ddl/) while taking care to update the xCluster replication configuration on both configurations involved.
+
+However, in some special cases, you need to follow different procedures.
+
```

Comment:
### Add a database to bidirectional xcluster replication

It is recommended that the databases should be added to bidirectional configurations right after they are created before any writes are performed on the tables within the dbs. 

To add a newly created database to bidirectional replication, follow the instructions at [Add a database to xcluster replication] in both directions.

Non-empty databases can still be added to replication in both directions but note that YBA will perform a [full copy](link to full copy section) to the target universe when it is added to the first direction, causing the current database on the target to be re-created.